spec.html

```diff
@@ -12605,6 +12605,7 @@
     </emu-clause>
   </emu-clause>
 
+  <!-- es6num="12.6" -->
```

Comment:
es6num is just a reference to the legacy section number in ES6. They should not be added or modified.